dashboard/app/assets/stylesheets/application.css.scss

```diff
@@ -215,6 +215,16 @@ img.video_thumbnail {
   }
 }
 
+.sign_in.show_only_on_mobile {
+  float: right;
+  margin-right: 10px;
+  margin-bottom: 10px;
+
+  @media screen and (max-device-height: 900px) and (max-device-width: 500px) {
```

Comment:
That's interesting.  You could, but only if you want to test that it works in all the right places.  If you want the more surgical fix, then matching the criteria sounds like the right thing to do.

Btw, have you tested at exactly 500px and 900px?  Sometimes we need to be off-by-one, depending whether the ranges are inclusive or exclusive.